bin/MadGraph5_aMCatNLO/cards/production/13TeV/HToZATo2L2B_ggfusion_b-associatedproduction/example_cards/HToZATo2L2B_500p00_300p00_20p00_bbH4F_TuneCP5_13TeV-amcatnlo_pythia8/HToZATo2L2B_500p00_300p00_20p00_bbH4F_TuneCP5_13TeV-amcatnlo_pythia8_run_card.dat

```diff
@@ -146,7 +148,7 @@ $DEFAULT_PDF_SETS = lhaid ! if pdlabel=lhapdf, this is the lhapdf number
 #***********************************************************************
   1.0	= jetalgo ! FastJet jet algorithm (1=kT, 0=C/A, -1=anti-kT)
   1.0	= jetradius ! The radius parameter for the jet algorithm
-  0.01	= ptj ! Min jet transverse momentum
+  10.	= ptj ! Min jet transverse momentum
```

Comment:
why is this relevant? there are no jets.

Reply:
This cut is still safe at the analysis level, we cut on jet pt > 20 GeV .
From my point of view, why do we generate events that we know we're going to cut!